front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -15,15 +15,30 @@ import useAuth from 'utils/hooks/useAuth';
 type ScenarioHeaderProps = {
   scenario: ScenarioResponse;
   infra: InfraWithState;
+  toggleTimeTable: () => void;
```

Comment:
Nit: "timetable" is an English word, so the second "T" shouldn't be capitalized.

Reply:
Sure, more like a miss due to quick typing :D thanks